app/_src/quickstart/universal-demo.md

```diff
@@ -21,51 +22,50 @@ demo-app --> redis
 ## Prerequisites
 
 * [Redis installed](https://redis.io/docs/getting-started/)
```

Comment:
I hit one thing that's not fluent here.
As a user, we may just start up the redis-server by the offical redis documentation with default `port 6379`. Later I feel a little bit confused with the DP startup command `servicePort: 26379`. However, go head with the latter doc section `Run kuma-counter-demo app` I realised that I need to start up the redis-server with the given command in port `26379`. 

So I was wondering could we put the redis-server startup command here?
> redis-server --port 26379 --daemonize yes && redis-cli -p 26379 set zone local

Reply:
I don't think this makes sense, this instruction does not say to startup redis just to install it, I can rephrase this prerequisite to be just `Redis CLI installed`

Reply:
`Redis-CLI` and `Redis-Server` installed